src/regions/ca/info/ca-qc/clinics.fr.md

```diff
@@ -14,6 +16,20 @@ Si vous croyez avoir une infection au COVID-19, vous pouvez contacter, sans frai
 
 [Source: Governement du Québec](https://santemontreal.qc.ca/en/public/coronavirus-covid-19/#c35268)
 
+#### Cliniques pour enfants
+
+Le CHU Sainte-Justine souhaite informer la population qu’il met en place un formulaire électronique visant à prioriser l’accès à un rendez-vous de sa clinique de dépistage de la COVID-19 aux enfants rentrant de voyage et présentant des symptômes.
+
+Ce formulaire en ligne est uniquement destiné aux familles dont les enfants reviennent de voyage et qui présentent des symptômes pouvant s’apparenter à ceux associés à la COVID-19, soit de la fièvre, des problèmes respiratoires et/ou de la toux.
+
+Pour obtenir un rendez-vous à la clinique de dépistage de la COVID-19 du CHU Sainte-Justine, visitez notre site Web pour [compléter le formulaire en ligne](https://www.chusj.org/fr/soins-services/C/COVID-19/Clinique-COVID-19?prov=news150320).
```

Comment:
I think this should say "visiter ce site web"? instead of "notre"

Reply:
good catch, fixing